### PR TITLE
Added code to check if channel has # in it and display validation err…

### PR DESCRIPTION
**Pull request by andrejonathan**
### Description
Added validation check to ensure # is not added in slack channel name

### Essential Checks:

* [x] Have you added tests where necessary?
* [x] Have you added metric gathering code where necessary (in `EventHandlers` and `CommandHandlers`)?
* [x] Have you added new commands to the displayable Help list?
* [x] Have you updated any necessary documentation?

### CODE REVIEW

**Comment by andrejonathan** [comment]
image

File: src/gluon/commands/team/LinkExistingTeamSlackChannel.ts [hunk 1]
```diff
@@ -33,7 +33,7 @@ export class LinkExistingTeamSlackChannel extends RecursiveParameterRequestComma
     @Parameter({
         required: true,
         displayName: "Team Slack Channel",
-        description: "The slack channel to link to your team.",
+        description: "The slack channel to link to your team (excluding the #)",
```
Comment (KieranHons) [inline src/gluon/commands/team/LinkExistingTeamSlackChannel.ts]: pattern: /^(?!#)/
validInput: "Please enter slack channel name without the #"

Comment (KieranHons) [reply src/gluon/commands/team/LinkExistingTeamSlackChannel.ts]: This will force the input to only be accepted if it doesn't start with a #

**Comment by andrejonathan** [comment]
image
Changed as suggested, thanks @KieranHons

File: src/gluon/commands/team/LinkExistingTeamSlackChannel.ts [hunk 2]
```diff
@@ -33,7 +33,9 @@ export class LinkExistingTeamSlackChannel extends RecursiveParameterRequestComma
...
+        pattern: /^(?!#)/,
+        validInput: "Please enter slack channel name without the #",
```
Comment (KieranHons) [inline src/gluon/commands/team/LinkExistingTeamSlackChannel.ts]:
```suggestion
        validInput: "a slack channel name without the #",
```

Comment (KieranHons) [reply src/gluon/commands/team/LinkExistingTeamSlackChannel.ts]: Just so when it's invalid its better worded.. must be Please enter slack channel etc is quite meh

**Comment by andrejonathan** [comment]
@KieranHons Please review

File: src/gluon/commands/team/LinkExistingTeamSlackChannel.ts (hunk 1 above)
Comment (andrejonathan) [reply src/gluon/commands/team/LinkExistingTeamSlackChannel.ts]: Done

File: src/gluon/commands/team/LinkExistingTeamSlackChannel.ts (hunk 2 above)
Comment (andrejonathan) [reply src/gluon/commands/team/LinkExistingTeamSlackChannel.ts]: Done

**Pull request merged**